### PR TITLE
py: REAL type tests for ARG_MAX and ARG_MIN aggregates

### PR DESCRIPTION
**Pull request by rivudhk**

### CODE REVIEW

File: python/tests/aggregate_tests/test_float_arg_max.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+from decimal import Decimal
+
+
+# REAL tests (considering precision of 7 digits)
+class aggtst_real_arg_max_value(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [{"c1": Decimal("57681.18"), "c2": Decimal("873315.8")}]
+        self.sql = """CREATE MATERIALIZED VIEW real_arg_max AS SELECT
+                      RF(ARG_MAX(c1, c2)) AS c1,
```
Comment (mihaibudiu) [inline python/tests/aggregate_tests/test_float_arg_max.py:11]: No need to use RF for DECIMAL, only for REAL tests.
In fact, something is fishy here, since the result is a DECIMAL, and it should be a REAL if you call RF.

**Comment by mihaibudiu** [comment]
what is the status of this PR?

File: python/tests/aggregate_tests/test_float_arg_max.py (hunk 1 above)
Comment (rivudhk) [reply python/tests/aggregate_tests/test_float_arg_max.py:11]: > In fact, something is fishy here, since the result is a DECIMAL, and it should be a REAL if you call RF.

@mihaibudiu since https://github.com/feldera/feldera/pull/3241 documents this edge case, we should be able to move forward with this PR.

Comment (mihaibudiu) [reply python/tests/aggregate_tests/test_float_arg_max.py:11]: this is wrong, I will file an issue.

**Comment by rivudhk** [comment]
The bug specified here is tracked by issue: #3396
